docs/en/deploy/Running-Alluxio-On-Kubernetes.md

```diff
@@ -687,6 +684,136 @@ If Alluxio has already been deployed with helm and now you want to enable FUSE,
 $ helm upgrade alluxio -f config.yaml --set fuse.enabled=true --set fuse.clientEnabled=true alluxio-charts/alluxio
 ```
 
+### Short-circuit Access
+
+Short-circuit access enables clients to perform read and write operations directly against the
+worker bypassing the networking interface.
+For performance-critical applications it is recommended to enable short-circuit operations
+against Alluxio because it can increase a client's read and write throughput when co-located with
+an Alluxio worker.
+
+#### Properties to Enable Short-Circuit Operations
+
+This feature is enabled by default, however requires extra configuration to work properly in
+Kubernetes environments.
+See sections below for how to configure short-circuit access.
+
+#### Disable Short-Circuit Operations
+To disable short-circuit operations, the operation depends on how you deploy Alluxio.
+
+> Note: As mentioned, disabling short-circuit access for Alluxio workers will result in 
+worse throughput 
```

Comment:
nit: "could result in worse I/O throughput" since the workload throughput might not be effected significantly